src/appleseed/renderer/kernel/lighting/sppm/sppmparameters.cpp

```diff
@@ -109,6 +109,7 @@ SPPMParameters::SPPMParameters(const ParamArray& params)
   , m_alpha(params.get_optional<float>("alpha", 0.7f))
   , m_max_photons_per_estimate(params.get_optional<size_t>("max_photons_per_estimate", 100))
   , m_dl_light_sample_count(params.get_optional<float>("dl_light_samples", 1.0))
+  , m_dl_low_light_threshold(params.get_optional<float>("dl_low_light_threshold", 0.0))
```

Comment:
Should be `0.0f` since it's a float (in fact, the line just above has the same problem).